### ISSUE TITLE
Form data doesn't show up in Google Sheets

### ISSUE THREAD

**Issue by angieherrera** [verbb/formie#488]
I submitted a support request last week [via the form on the Formie page](https://verbb.io/craft-plugins/formie/support) and haven't heard back. So, trying here.

**Description**
I've set up Google Sheets integration in Formie which is connected and enabled. As far as I can tell, that part is all set. When I submit one of the forms we have setup with that integration, the data never shows up in the spreadsheet.

I've looked in formie.log and web.log and can't spot anything obvious to me. Not sure what else needs adjusting or how to fix.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.4
- Craft version: 3.6.15
- Multi-site: No

**Comment by engram-design** [comment]
I'm afraid that support request didn't come through, I'll look what happened there (the site doesn't use Formie itself yet!!). You're welcome to email us directly at `web at verbb.io`.

Difficult to tell without seeing what you're mapping to, and what sort of fields they are. I assume the submissions are coming through and available in the control panel? Do email notifications also come through (if enabled)?

**Comment by angieherrera** [comment]
No worries; will send an email with additional info!

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.6](https://github.com/verbb/formie/releases/tag/1.4.6)
